addons/pad/static/tests/pad_tests.js

```diff
@@ -307,4 +307,81 @@ QUnit.module('pad widget', {
         delete FieldPad.prototype.isPadConfigured;
     });
 
+    QUnit.test('Quick Edition: click on the pad', async function (assert) {
+        assert.expect(2);
+
+        const form = await createView({
+            View: FormView,
+            model: 'task',
+            data: this.data,
+            arch: `
+                <form>
+                    <sheet>
+                        <group>
+                            <field name="description" widget="pad"/>
+                        </group>
+                    </sheet>
+                </form>`,
+            res_id: 2,
+            mockRPC: function (route, args) {
+                if (!args.method) {
```

Comment:
if there is no method, use the route, otherwise we don't know what you're mocking